src/vos/tests/evt_ctl.c

```diff
@@ -244,8 +245,8 @@ bio_alloc_init(struct utest_context *utx, bio_addr_t *addr, const void *src,
 	if (src == NULL) {
 		addr->ba_hole = 1;
 		return 0;
-	}
-
+	} else
```

Comment:
When the if has brackets, the else should match it and also have brackets...checkpatch will not complain in this case.

Reply:
Done. Should be in the next patch...